releasenotes/notes/agent-sidecar-security-cbfd5ea9f72124d0.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+enhancements:
+  - |
+    The Datadog Cluster Agent admission controller agent sidecar injection now sets up
```

Comment:
```suggestion
    The Datadog Cluster Agent admission controller Agent sidecar injection now sets up
```

Reply:
Or leave these lowercase if you're referring to "agent" in a more general sense, not the Datadog Agent.